src/main/java/io/github/darealturtywurty/turtychemistry/mixin/MixinHooks.java

```diff
@@ -0,0 +1,6 @@
+package io.github.darealturtywurty.turtychemistry.mixin;
+
+public final class MixinHooks {
+
+
+}
```

Comment:
This should not be in the `mixin` package. It should be in `core`. Also it should have the hooks in it.

Reply:
what are the hooks?

Reply:
The mixin methods.